service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/OrdersNeededControllerTest.java

```diff
@@ -130,4 +131,15 @@ void shouldSetOrdersToOrdersSolicitorForSolicitorsAfterOrderSubmitted() {
         assertThat(response.getData().get("representativeType")).isEqualTo("RESPONDENT_SOLICITOR");
         assertThat(response.getData().get("orders")).isEqualTo(response.getData().get("ordersSolicitor"));
     }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    void shouldEducationSupervisionOrderDataWhenEducationSupervisionOrderIsUnselected() {
```

Comment:
Maybe a typo here, should this be `shouldRemoveEducationSupervisionOrderDataWhenEducationSupervisionOrderIsUnselected()`